src/Application/State.elm

```diff
@@ -184,24 +183,27 @@ update msg =
         Select a ->
             Drive.select a
 
-        ShowPreviewOverlay ->
-            Drive.showPreviewOverlay
-
         ShowRenameItemModal a ->
             Drive.showRenameItemModal a
 
         ToggleExpandedSidebar ->
             Drive.toggleExpandedSidebar
 
-        ToggleSidebarMode a ->
-            Drive.toggleSidebarMode a
+        ToggleSidebarAddOrCreate ->
+            Drive.toggleSidebarAddOrCreate
+
+        SidebarMsg sidebarMsg ->
+            Drive.updateSidebar sidebarMsg
 
         -----------------------------------------
         -- File System
         -----------------------------------------
         GotFsDirectoryList a ->
             FileSystem.gotDirectoryList a
 
+        GotItemUtf8 a ->
```

Comment:
Nitpick 😅 Can you rename this to `GotFsItemUtf8` and put it an the last item under 'file system' (alphabetically). Just to be consistent.

Reply:
Yep!